File: TransCoder_tokenized_test_set_functions/python/DYNAMIC_PROGRAMMING_HIGH_EFFORT_VS_LOW_EFFORT_TASKS_PROBLEM.py
def maxTasks ( high, low, n ) :
    if ( n <= 0 ) :
        return 0
    return max ( high [ n - 1 ] + maxTasks ( high, low, ( n - 2 ) ), low [ n - 1 ] + maxTasks ( high, low, ( n - 1 ) ) );

File: results/py/DYNAMIC_PROGRAMMING_HIGH_EFFORT_VS_LOW_EFFORT_TASKS_PROBLEM.cpp
int f_filled ( int * high , int * low , int n ) {
  if ( ( n <= 0 ) || ( n == 1 ) ) return 0 ;
  return max ( high [ n - 1 ] + f_filled ( high , low , ( n - 2 ) ) , low [ n - 1 ] + f_filled ( high , low , ( n - 1 ) ) ) ;
}
